ui/shared/tabs/failureSummary/FailureSummaryTab.jsx

```diff
@@ -86,11 +86,11 @@ class FailureSummaryTab extends React.Component {
         if (suggestion.bugs.open_recent.length > 0) {
           suggestion.bugs.open_recent.forEach((bug) => {
             if (bug.summary.endsWith('single tracking bug')) {
-              simpleCase = [bug];
+              simpleCase.push(bug);
             }
           });
         }
-        if (suggestion.bugs.all_others.length > 0) {
+        if (simpleCase.length == 0 && suggestion.bugs.all_others.length > 0) {
           suggestion.bugs.all_others.forEach((bug) => {
             if (bug.summary.endsWith('single tracking bug')) {
               simpleCase = [bug];
```

Comment:
should this be push also?

Reply:
Have also converted this now. The main concern was recent bugs including two such getting filed in a short time before the bug cache in TH got updated.